src/Components/Intercom.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import * as Scrivito from 'scrivito';
+import Helmet from 'react-helmet';
+
+class Intercom extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isIntercomInstalled: false,
+    };
+  }
+
+  componentDidMount() {
+    Scrivito.load(() => getIntercomAppId()).then(intercomAppId => {
```

Comment:
* [ ] FYI: the anonymous function indirection is not necessary. `Scrivito.load(getIntercomAppId).then`